### ISSUE TITLE
StaticElement from schema with content as function ignores tag field

### ISSUE THREAD

**Issue by thely** [vueform/vueform#229]
### Environment

Vite 5.1.5, Vue 3.4.21

### Reproduction

https://stackblitz.com/edit/github-or77kl?file=src%2FApp.vue

### Describe the bug

If you use StaticElement as a schema but try to set the content as a function, the tag field is then ignored, and no tag is applied.

```html
<script setup>
import { reactive } from 'vue';
const schema = reactive({
  header: {
    type: 'static',
    tag: 'h2',
    content: ($el) => {
      return 'Hello World';
    },
  },
  hello_world: {
    type: 'text',
    label: 'Hello',
    placeholder: 'World',
  },
});
</script>

<template>
  <div class="max-w-[500px] mx-auto mt-10">
    <Vueform :schema="schema"> </Vueform>
  </div>
</template>

<style></style>
```

**Comment by thely** [comment]
Sorry, yes - while I did close out the reactive early on StackBlitz, in your link, the StaticElement is still not being enclosed in an `h2`. This seems to happen only when the StaticElement's content is a function rather than a static value. For example, if I change it to the below code, the tag field works again.

```javascript
header: {
  type: 'static',
  tag: 'h2',
  content: 'Hello World'
}
```

**Comment by adamberecz** [comment]
Should be now fixed in `1.9.7`.
